blocks/article-list/article-list.js

```diff
@@ -13,25 +15,29 @@ const ARTICLE_FORMATTER = new Intl.DateTimeFormat('default', {
 });
 
 // TODO: change to web component once available
-function renderCard(card) {
+async function renderCard(card) {
+  const placeholders = await fetchPlaceholders();
   const formattedDate = ARTICLE_FORMATTER.format(new Date(card.publicationDate * 1000));
   const cardclass = `card${card['hot story'] ? ' hot-story' : ''}`;
   const cardAuthorUrl = `/author/${toClassName(card.author).replace('-', '')}`; // TODO look up author URL from index
+  const showArticleTemplate = placeholders.showArticleTypeOnArticleCards === 'yes';
```

Comment:
Instead of using placeholders here I suggest to move this check to the block calling the `renderCard()` function and adding this as card option.

Reply:
hm, but the article cards are created automatically e.g. for the related articles.
I was thinking we may want a site wide toggle for this as so far we're not fully clear if we'll import just the blogs or all types. wdyt?

Reply:
I think there a multiple aspects here, sorry if I take this as an example:

- using placeholders as "config" is not best practice, I know it is done in other projects
- fetching the placeholder inside `rendereCard` will call fetch every time a card is rendered, this is even with cached placeholder.json not effective and adds CPU cycles
- in my opinion renderCard should be self contained and get all data & config options from the blocks using it
- lets only build stuff we really need, is there a use-case for the blog only to have this configurable? If no lets not add this feature. Is this needed maybe later for entire ContentHub, yes maybe. Lets add it once we need it.

Reply:
understood and agreed. we'll just change the code when the deicision changes, fine by me